### DISCUSSION TITLE
Python, API, Docker

### DISCUSSION THREAD

**Discussion by devminjae** [Korean-American-IT-Association-KAITA/kaita-project-management#43] [5. 트레이닝/멘토링 이벤트]
pre-requirement: python에 대한 기본 지식(at least function)
Introduction: python의 library중 하나인 Flask를 이용해서 API를 구성, docker로 자신의 이미지를 구성해서 특정 port로 배포하기.

Python으로 구현할 내용은 NLP중 하나인 fuzzywuzzy를 이용해서 library에 등록(ADD)/수정(POST)/삭제(DEL)/업데이트(UPDATE)/similarity top 10(GET)을 할 수 있는 api를 만들어 배포하는 과정입니다. 데이터는 text혹은 nosql로 저장할 예정입니다.
프로젝트의 자세한 내용은 추후 변경될 가능성이 있습니다. 

* docker는 기본적으로 컴퓨터에 설치되어 있는 것으로 가정하고 시작합니다. 

순서
1. docekr를 이용한 python 환경만들기(jupyterlab)
2. notebook을 이용해서 function test and file로 이동
3. package배포 setup 및 package설치
4. package를 api로 감싸기(Flask)
5. API를 docker에 배포 및 테스트

CST기준 금요일 or 토요일 10시에 진행할 예정이고 1 session으로 부족할 시 2session으로 나눠서 할 예정입니다.

**Comment by kikibell1004** [comment]
미팅 참석 하고 싶습니다. 미팅 Invitation 보내주실 예정이신가요?

**Comment by Indosaram** [comment]
저도 등록합니다!
